basic_pipeline/08_Depayloader.md

```diff
@@ -1,6 +1,6 @@
 # Depeyloader
 
-Since we have packets put in order by the Ordering Buffer, we can assemble them into the original frames.
+Since we have [packets](../glossary/glossary.md#packet) put in order by the [Ordering Buffer](../glossary/glossary.md#jitter-buffer--ordering-buffer), we can assemble them into the original [frames](../glossary/glossary.md#frame).
```

Comment:
In fact, we are talking here about our implementation of the Ordering Buffer, so probably it would make more sense to link to the 6th chapter instead of the Jitter Buffer definition